### ISSUE TITLE
Python codegen examples missing from wizard

### ISSUE THREAD

**Issue by gabewillen** [Yakindu/examples#211]
Am I doing something wrong the Python example doesn't show up in the default examples wizard or the extended BETA tools.

![image](https://user-images.githubusercontent.com/17092234/92401358-1d72bf80-f0f3-11ea-815d-cd45a4572b1f.png)

**Comment by tkutz** [comment]
We separated the development of the examples onto two branches in order to have one version of the examples that still works with the 3.x Statechart Tools, and one that is already updated to the 4.x Statechart Tools.

The last released 4.x Beta version still has the example branch set to "release", which is the 3.x version of the examples now. There we have removed the Python examples as we moved the Python generator to the 4.x version and will no more maintain/deliver it with the 3.x version.

For your 4.x Statechart Tools we have an own example branch which we will release together with the new 4.x version which will come out soon. There you will also find the python examples. But until then you will have to live without the examples. Sorry for the inconvenience.

And there is always our documentation which should help setting up the python generator on your own:
https://www.itemis.com/en/yakindu/state-machine/documentation/user-guide/codegen_python_code_generator
